scriptmodules/supplementary/usbromservice.sh

```diff
@@ -41,10 +41,19 @@ function install_bin_usbromservice() {
 
 function install_scripts_usbromservice() {
     # copy our mount.d scripts over
+    local filesrc
     local file
     local dest
-    for file in "$md_data/"*; do
-        dest="/etc/usbmount/mount.d/${file##*/}"
+    for file in "$md_data/"*.mount"; do
+	filesrc=${file%%.mount}
+        dest="/etc/usbmount/mount.d/${filesrc##*/}"
+        sed "s/USERTOBECHOSEN/$user/g" "$file" >"$dest"
+        chmod +x "$dest"
+    done
+
+    for file in "$md_data/"*.umount"; do
+	filesrc=${file%%.umount}
+        dest="/etc/usbmount/umount.d/${filesrc##*/}"
```

Comment:
why the renaming to .mount - I would have though a single copy loop would be possible for copying all files ?

Reply:
I see the reasoning, but it might be better to just use a different structure to simplify. Is there much use in the umount script ?